docs/source/keylime/index.rst

```diff
@@ -0,0 +1,135 @@
+Keylime
+=======
+
+ESI provides a guide to use `Keylime`_ outside Ironic for run-time attestation with `IMA`_.
+
+Prerequisites
+-------------
+* A private attestation network.
+* A node specified as Keylime server.
+* The nodes-to-be-attested with an active TPM 2.0 and keylime agent installed.
+
+Setup
+-----
+
+Build Keylime Server
+~~~~~~~~~~~~~~~~~~~~
+Provision the node:
+
+.. prompt:: bash $
+
+  metalsmith deploy --image <image> --network <network> --resource-class baremetal --ssh-public-key <path_to_public_key>
+
+* Note: the server needs to be connected to the public internet in order to download Keylime packages. Attaching the node to the public network could be as simple as running the command ``openstack esi node network attach (--network <network> | --port <port>) <node>``.
+
+Install Keylime packages:
+
+* After the node is active, ssh into it. There are various ways to acquire Keylime. One possible way is listed below:
+
+  .. prompt:: bash $
+
+    git clone https://github.com/keylime/keylime.git
+    cd keylime && ./installer.sh
+
+Modify /etc/keylime.conf:
+
+* Set ``require_ek_cert`` to false if the physical TPM does not have EK certificates.
+* Set the binding address for verifier and registrar, for example:
+
+  .. prompt::
+
+    [cloud_verifier]
+    cloudverifier_ip = 0.0.0.0
+
+    [registrar]
+    registrar_ip = 0.0.0.0
+
+Start Keylime services:
+
+.. prompt:: bash $
+
+  keylime_verifier
+  keylime_registrar
+
```

Comment:
And here, specify that if the node is not on the attestation network, it should now be put there using 'openstack esi node network attach --network <attestation_network> <node>'